Nodejs/Product/Nodejs/visualstudio_nodejs_repl.js

```diff
@@ -120,8 +122,6 @@ function processRequest(command) {
                             throw err;
                         }
                     }
-                } else {
-                    obj = vm.runInContext(evalPrefix + '(' + command["code"] + ')', context);
```

Comment:
Looks like the parens were introduced in #876 - do you remember if there was a reason for them?